docs/CODE_GUIDELINES.md

```diff
@@ -447,6 +486,23 @@ class Example
 }
 ```
 
+### 7.2. Use C++ wrappers for C headers
+To use C symbols use C++ wrappers headers, by using the `std` namespace prefix.
+
+✅ Good:
+```cpp
+#include <cstring>
+[...]
+std::strcpy(destText, sourceText);
```

Comment:
can we have another example maybe? `strcpy` is unsafe and should never ever be used.

Reply:
changed with `strlen`